TestSave/all_info_present fails with release 2.5.10

**Describe the bug**
Hey! I maintain the [slackdump AUR package](https://aur.archlinux.org/packages/slackdump) for Arch Linux (thanks for providing this app btw 😉), and I was going to bump the version to the latest (2.5.9 -> 2.5.10) but I noticed that one of the tests fails now.

This is what I get when I run `go test ./...`:

```
--- FAIL: TestSave (0.00s)
    --- FAIL: TestSave/all_info_present (0.00s)
        auth_test.go:112: Save() = {"Token":"token_value","Cookie":[{"Name":"d","Value":"abc","Quoted":false,"Path":"","Domain":"","Expires":"0001-01-01T00:00:00Z","RawExpires":"","MaxAge":0,"Secure":false,"HttpOnly":false,"SameSite":0,"Partitioned":false,"Raw":"","Unparsed":null}]}
            , want {"Token":"token_value","Cookie":[{"Name":"d","Value":"abc","Path":"","Domain":"","Expires":"0001-01-01T00:00:00Z","RawExpires":"","MaxAge":0,"Secure":false,"HttpOnly":false,"SameSite":0,"Raw":"","Unparsed":null}]}
FAIL
```

I seems that the two versions differ in the `"Quoted":false` and the `"Partitioned":false`.

**To Reproduce**
Steps to reproduce the behavior:
1. Download latest tarball
2. Compile with `go build -o build ./cmd/...` (if it matters, I use this `GOFLAGS="-buildmode=pie -trimpath -ldflags=-linkmode=external -mod=readonly -modcacherw"`)
3. Run tests with `go test ./...`

**Expected behavior**
All tests pass.

**Output**
Already included above

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - OS Version: -
 - Slackdump Version: 2.5.10

--- a/.github/workflows/go.yml
+++ b/.github/workflows/go.yml
@@ -11,12 +11,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
 
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v5
       with:
-        go-version: "1.21"
+        go-version: "1.23"
 
     - name: Build
       run: go build -v ./...
@@ -24,6 +24,9 @@ jobs:
     - name: Check Catalogue
       run: |
         BASE_DIR="contrib" go run ./contrib/_gen -v
+
+    - name: AUR build test
+      run: make aurtest
 
     - name: Test
       run: make test

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -18,7 +18,7 @@ jobs:
       - run: git fetch --force --tags
       - uses: actions/setup-go@v5
         with:
-          go-version: '>=1.19.5'
+          go-version: '>=1.23.0'
           cache: true
       # More assembly might be required: Docker logins, GPG, etc. It all depends
       # on your needs.

--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@ clean:
 
 test:
 	go test -race -cover -count=3 ./...
-aurtest: test
+aurtest:
 	GOFLAGS="-buildmode=pie -trimpath -ldflags=-linkmode=external -mod=readonly -modcacherw" go build -o 'deleteme' ./cmd/...
 	rm deleteme
 .PHONY: aurtest

--- a/Makefile
+++ b/Makefile
@@ -54,6 +54,10 @@ clean:
 
 test:
 	go test -race -cover -count=3 ./...
+aurtest: test
+	GOFLAGS="-buildmode=pie -trimpath -ldflags=-linkmode=external -mod=readonly -modcacherw" go build -o 'deleteme' ./cmd/...
+	rm deleteme
+.PHONY: aurtest
 
 docker_test:
 	docker build .

--- a/auth/auth_test.go
+++ b/auth/auth_test.go
@@ -69,7 +69,7 @@ func TestSave(t *testing.T) {
 			args{ValueAuth{simpleProvider{Token: "token_value", Cookie: []*http.Cookie{
 				{Name: "d", Value: "abc"},
 			}}}},
-			`{"Token":"token_value","Cookie":[{"Name":"d","Value":"abc","Path":"","Domain":"","Expires":"0001-01-01T00:00:00Z","RawExpires":"","MaxAge":0,"Secure":false,"HttpOnly":false,"SameSite":0,"Raw":"","Unparsed":null}]}` + "\n",
+			`{"Token":"token_value","Cookie":[{"Name":"d","Value":"abc","Quoted":false,"Path":"","Domain":"","Expires":"0001-01-01T00:00:00Z","RawExpires":"","MaxAge":0,"Secure":false,"HttpOnly":false,"SameSite":0,"Partitioned":false,"Raw":"","Unparsed":null}]}` + "\n",
 			false,
 		},
 		{

--- a/go.mod
+++ b/go.mod
@@ -1,8 +1,6 @@
 module github.com/rusq/slackdump/v2
 
-go 1.22
-
-toolchain go1.22.1
+go 1.23
 
 require (
 	github.com/MercuryEngineering/CookieMonster v0.0.0-20180304172713-1584578b3403

